tests/cmds/test_show.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.full_block import FullBlock
+from tests.cmds.cmd_test_utils import TestFullNodeRpcClient, TestRpcClients, cli_assert_shortcut, run_cli_command
+from tests.cmds.testing_classes import height_hash
+from tests.util.test_full_block_utils import get_full_blocks
+
+BASE_LIST = ["show"]
+
+
+@dataclass
+class ShowFullNodeRpcClient(TestFullNodeRpcClient):
+    full_block: FullBlock = field(init=False)
+
+    def __post_init__(self) -> None:
+        full_block: Optional[FullBlock] = None
+        for f_b in get_full_blocks(tx_block=True):
+            if f_b is not None and f_b.foliage_transaction_block is not None:
+                full_block = f_b
+                break
+        assert full_block is not None
+        self.full_block = full_block
```

Comment:
this is just generating a random FullBlock, right? I think this would be clearer as a free function